pfb-analysis/import/import_neighborhood.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 NB_INPUT_SRID="${NB_INPUT_SRID:-4326}"
-NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-4326}"
-NB_BOUNDARY_BUFFER="${NB_BOUNDARY_BUFFER:-0}"
+NB_OUTPUT_SRID="${NB_OUTPUT_SRID:-2163}"
```

Comment:
I'm not sure we want this to default to MA state plane, as that's an odd choice. Should probably still default to 4326 as its a generic projection that would at least (sort of) work for most locations.